packages/components/src/combobox-control/index.js

```diff
@@ -30,15 +33,19 @@ export default function ComboboxControl( {
 		highlightedIndex,
 		selectedItem,
 	} = useCombobox( {
-		initialSelectedItem: items[ 0 ],
+		initialSelectedItem: items[ initialHighlightedIndex || 0 ],
```

Comment:
This contradicts the Downshift API. If this is the desired behavior, then you need an `initialSelectedItem` prop, not `initialHighlightedIndex`.

Reply:
I will try using `initialSelectedItem` directly again.

Reply:
improved with `initialSelectedItem` in https://github.com/WordPress/gutenberg/pull/23237/commits/d700436a80faba003c4cbb7b60f3f64b5d19f88. Also fixed the comparison so objects work and the current selected author is now correctly marked with a checkmark in the dropdown.